Source/Core/Core/CoreParameter.cpp

```diff
@@ -420,10 +420,13 @@ IniFile SCoreStartupParameter::LoadGameIni(const std::string& id, int revision)
 void SCoreStartupParameter::LoadGameIni(IniFile* game_ini, const std::string& path,
                                         const std::string& id, int revision)
 {
-	// INIs that match all regions
+	// INIs that match all regions (Without publisher codes)
 	if (id.size() >= 4)
 		game_ini->Load(path + id.substr(0, 3) + ".ini", true);
 
+	// INIs that match all regions (With publisher codes)
+	game_ini->Load(path + id.substr(0, 3) + "-" + id.substr(4) + ".ini", true);
```

Comment:
Thinking about it, the only scenarios I can think of where a game's ID wouldn't be 6 digits is if the game doesn't have a proper GameID (NDDemo/Peach's Castle), or if we ever switched to 4 digit IDs for WiiWare titles. We could probably do a check for `if (id.size() == 6`